src/please.go

```diff
@@ -925,6 +928,14 @@ func (overrides ConfigOverrides) Complete(match string) []flags.Completion {
 	return core.DefaultConfiguration().Completions(match)
 }
 
+// Get an absolute path from a relative path.
+func getAbsolutePath(path string, here string) string {
+	if path[0] != '/' && path[0] != '~' {
```

Comment:
I think `~` is expanded by bash (or similar)  so the latter check is probably not necessary.

Reply:
I've had issues before where the shell wasn't expanding `~`, so it might be better to take it into account.